### ISSUE TITLE
Widget header text format. Show current week number, Widget name...

### ISSUE THREAD

**Issue by val24** [plusonelabs/calendar-widget#347]
Hello,

I would find useful to have the current week number in the widget header (I thought it had already been mentionned, but I can't find the issue any more).

Thanks

**Comment by yvolk** [comment]
In order to avoid work duplication I think that we will reuse Day header format for the Widget header. 
So this will be implemented after #216

**Comment by yvolk** [comment]
I created separate ticket andstatus#7 for a reusable Preference dialog for date format that we could use for setting Date format of the Widget header also.

**Comment by yvolk** [comment]
Beta version with Date formatting preference released. See https://github.com/andstatus/todoagenda/issues/7 and download from here: https://github.com/andstatus/todoagenda/issues/3

**Comment by yvolk** [comment]
Tip: in order to have any text in the widget header, you can add any text in single quotes, e.g. like this:
'Test:' bbbb EEE, d MMM

**Comment by yvolk** [comment]
Released in v.4.0.6
